llvm/lib/SYCLLowerIR/ESIMD/ESIMDVerifier.cpp

```diff
@@ -187,4 +187,6 @@ INITIALIZE_PASS_BEGIN(ESIMDVerifier, DEBUG_TYPE, "ESIMD-specific IR verifier",
 INITIALIZE_PASS_END(ESIMDVerifier, DEBUG_TYPE, "ESIMD-specific IR verifier",
                     false, false)
 
-ModulePass *llvm::createESIMDVerifierPass() { return new ESIMDVerifier(); }
+ModulePass *llvm::createESIMDVerifierPass(bool MayNeedForceStatelessMemModeAPI) {
```

Comment:
Before this PR: Creation of ESIMDVerifier() without arguments left the field 'ForceStatelessMem' uninitialized and thus having random value.